android-components/components/browser/state/src/main/java/mozilla/components/browser/state/engine/middleware/TranslationsMiddleware.kt

```diff
@@ -62,6 +66,11 @@ class TranslationsMiddleware(
                             requestPageSettings(context, action.tabId)
                         }
                     }
+                    TranslationOperation.FETCH_AUTOMATIC_LANGUAGE_SETTINGS -> {
```

Comment:
Manual action would only be used if something went wrong during `init` or to force a refresh. (Page settings and global language settings are intertwined.) 

I plan to move the current `neverTranslateSites` off of the session store and onto the global store too in [1881611](https://bugzilla.mozilla.org/show_bug.cgi?id=1881611). (Originally made sense because of the intertwining with the `pageSettings`, but I think long term, it would be better on the global object for an easier `init`.)